pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/TrainerCreateDTO.java

```diff
@@ -0,0 +1,85 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dto;
+
+import static cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerDTOConstants.*;
+import java.time.LocalDate;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * DTO class for create Trainer.
+ *
+ * @author Jiří Medveď 38451
+ */
+public class TrainerCreateDTO {

```

Comment:
Missing equals, hashCode and toString methods, auto generated with all properties is completely enough.